test/Core.Test/AdminConsole/AutoFixture/OrganizationFixtures.cs

```diff
@@ -19,17 +19,24 @@ public class OrganizationCustomization : ICustomization
 {
     public bool UseGroups { get; set; }
     public bool FlexibleCollections { get; set; }
+    public PlanType PlanType { get; set; }
 
     public void Customize(IFixture fixture)
     {
         var organizationId = Guid.NewGuid();
         var maxCollections = (short)new Random().Next(10, short.MaxValue);
+        var plan = StaticStore.Plans.FirstOrDefault(p => p.Type == PlanType);
+        var seats = (short)new Random().Next(plan.PasswordManager.BaseSeats, plan.PasswordManager.MaxSeats ?? short.MaxValue);
+        var smSeats = (short)new Random().Next(plan.SecretsManager.BaseSeats, plan.SecretsManager.MaxSeats ?? short.MaxValue);
```

Comment:
`plan.SecretsManager` can be null if the plan doesn't offer it - e.g. families. This will work fine with the default (Enterprise) but potentially not otherwise.

Suggestion only:

```cs
var smSeats = plan.SupportsSecretsManager 
  ? (short)new Random().Next(plan.SecretsManager.BaseSeats, plan.SecretsManager.MaxSeats ?? short.MaxValue) 
  : null;
```

Reply:
Big oversight on my part, thanks for spotting this!